docker-compose.yml

```diff
@@ -91,6 +92,7 @@ services:
     - NET_ADMIN
     - AUDIT_WRITE
     - SYS_CHROOT
+    #- SYS_RAWIO # Only needed if you require access to real devices like FIDO2 tokens
```

Comment:
If this can not work on its own, I think we can remove this patch.

Reply:
I removed this patch